packages/flutter/lib/src/widgets/form.dart

```diff
@@ -332,6 +349,17 @@ class FormField<T> extends StatefulWidget {
   ///    Flutter.
   final String? restorationId;
 
+  /// submission key used to identify this [FormField] value at the [Form] level
+  ///
+  /// submissionKey must be set to a non null value for the this [FormField]
+  /// value to be included in the parent's [Form] submit.
+  ///
+  /// See also:
+  ///
+  ///  * [FormState], which explains the submission behaviour in its submit
```

Comment:
Let's reference the method name as well

```suggestion
  ///  * [FormState.submit], which explains the submission behaviour in its submit
```